sdk/include/xdk/ntbasedef.h

```diff
@@ -788,7 +788,7 @@ extern "C++" { \
 #endif /* _M_AMD64 */
 
 /* C_ASSERT Definition */
-#define C_ASSERT(expr) extern char (*c_assert(void)) [(expr) ? 1 : -1]
+#define C_ASSERT(expr) typedef char C_ASSERT_failed_Please_check_expr_is_TRUE[(expr) ? 1 : -1]
```

Comment:
I think it would be better to just copy those already-existing definitions, see e.g.
https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/include/psdk/intsafe.h;hb=072503c1fc133f48c4d710362b42b4cc4fb7f192#l70

Reply:
Fixed.